diff --git a/games/stockfish/Makefile b/games/stockfish/Makefile
index 77f2817..5ac5cd7 100644
--- a/games/stockfish/Makefile
+++ b/games/stockfish/Makefile
@@ -1,6 +1,6 @@
 PORTNAME=	stockfish
 DISTVERSIONPREFIX=	sf_
-DISTVERSION=	14
+DISTVERSION=	15.1
 CATEGORIES=	games
 MASTER_SITES=	https://tests.stockfishchess.org/api/nn/:nnue
 DISTFILES=	nn-${_NNUE_VER}.nnue:nnue
@@ -25,7 +25,7 @@ MAKE_ARGS=	ARCH=${MYARCH} \
 		PREFIX=${PREFIX}
 ALL_TARGET=	build
 
-_NNUE_VER=	3475407dc199
+_NNUE_VER=	ad9b42354671
 
 WRKSRC_SUBDIR=	src
 
diff --git a/games/stockfish/distinfo b/games/stockfish/distinfo
index fd193eb..f3ff5c1 100644
--- a/games/stockfish/distinfo
+++ b/games/stockfish/distinfo
@@ -1,5 +1,5 @@
-TIMESTAMP = 1629298350
-SHA256 (nn-3475407dc199.nnue) = 3475407dc19973ea44467678634cce023d620e419770c111cc8937fe6689ec87
-SIZE (nn-3475407dc199.nnue) = 47721371
-SHA256 (official-stockfish-Stockfish-sf_14_GH0.tar.gz) = 6f35e3e684da87d27d3f29ec7281ac81468a5a86b4d99ac5c599addc984a766c
-SIZE (official-stockfish-Stockfish-sf_14_GH0.tar.gz) = 183879
+TIMESTAMP = 1670387780
+SHA256 (nn-ad9b42354671.nnue) = ad9b423546714137916bd38978af6fd68d7b8951bef25ff76bf43da72d6cb786
+SIZE (nn-ad9b42354671.nnue) = 47001499
+SHA256 (official-stockfish-Stockfish-sf_15.1_GH0.tar.gz) = d4272657905319328294355973faee40a8c28e3eecb0e7b266ed34ff33383b76
+SIZE (official-stockfish-Stockfish-sf_15.1_GH0.tar.gz) = 193408
