diff --git a/ports/games/stockfish/Makefile b/ports/games/stockfish/Makefile
index ec39fed..434a92c 100644
--- a/ports/games/stockfish/Makefile
+++ b/ports/games/stockfish/Makefile
@@ -1,6 +1,6 @@
 PORTNAME=	stockfish
 DISTVERSIONPREFIX=	sf_
-DISTVERSION=	16.1
+DISTVERSION=	17
 CATEGORIES=	games
 MASTER_SITES=	https://tests.stockfishchess.org/api/nn/:nnue1 \
 		https://tests.stockfishchess.org/api/nn/:nnue2
@@ -10,7 +10,7 @@ EXTRACT_ONLY=	${DISTNAME}${EXTRACT_SUFX}
 
 MAINTAINER=	execve@gmail.com
 COMMENT=	Open source chess engine
-WWW=		https://www.stockfishchess.com/
+WWW=		https://www.stockfishchess.org/
 
 LICENSE=	GPLv3
 LICENSE_FILE=	${WRKSRC}/../Copying.txt
@@ -27,8 +27,8 @@ MAKE_ARGS=	ARCH=${MYARCH} \
 		PREFIX=${PREFIX}
 ALL_TARGET=	build
 
-_NNUE1_VER=	b1a57edbea57
-_NNUE2_VER=	baff1ede1f90
+_NNUE1_VER=	1111cefa1111
+_NNUE2_VER=	37f18f62d772
 
 BINARY_ALIAS=	wget=true
 
diff --git a/ports/games/stockfish/distinfo b/ports/games/stockfish/distinfo
index 8b0ca3b..4aa16f1 100644
--- a/ports/games/stockfish/distinfo
+++ b/ports/games/stockfish/distinfo
@@ -1,7 +1,7 @@
-TIMESTAMP = 1709006625
-SHA256 (nn-b1a57edbea57.nnue) = b1a57edbea574ca8b88d6837473845791beb53d885f87f86d5ccdd5659fbf3b2
-SIZE (nn-b1a57edbea57.nnue) = 65429575
-SHA256 (nn-baff1ede1f90.nnue) = baff1ede1f90c1dd1b4f772f1eff29848821801e8186345da7f0eb4121bd6f63
-SIZE (nn-baff1ede1f90.nnue) = 3480122
-SHA256 (official-stockfish-Stockfish-sf_16.1_GH0.tar.gz) = a5f94793b5d4155310397ba89e9c4266570ef0f24cd47de41a9103556f811b82
-SIZE (official-stockfish-Stockfish-sf_16.1_GH0.tar.gz) = 176846
+TIMESTAMP = 1725906819
+SHA256 (nn-1111cefa1111.nnue) = 1111cefa11116b77161bd4b14dab4c50f26e5920c756f4861592be3dcd6de174
+SIZE (nn-1111cefa1111.nnue) = 74874478
+SHA256 (nn-37f18f62d772.nnue) = 37f18f62d772f3107e1d6aaca3898c130c3c86f2ab63e6555fbbca20635a899d
+SIZE (nn-37f18f62d772.nnue) = 3519630
+SHA256 (official-stockfish-Stockfish-sf_17_GH0.tar.gz) = 8f9b52285c3348c065b7cb58410626df16d7416a2e60a3b04f3ec7c038e67ad1
+SIZE (official-stockfish-Stockfish-sf_17_GH0.tar.gz) = 204647
