diff --git a/games/stockfish/Makefile b/games/stockfish/Makefile
index 069e778..ec1ddd6 100644
--- a/games/stockfish/Makefile
+++ b/games/stockfish/Makefile
@@ -1,11 +1,9 @@
 PORTNAME=	stockfish
 DISTVERSIONPREFIX=	sf_
-DISTVERSION=	18
+DISTVERSION=	19
 CATEGORIES=	games
-MASTER_SITES=	https://tests.stockfishchess.org/api/nn/:nnue1 \
-		https://tests.stockfishchess.org/api/nn/:nnue2
-DISTFILES=	nn-${_NNUE1_VER}.nnue:nnue1 \
-		nn-${_NNUE2_VER}.nnue:nnue2
+MASTER_SITES=	https://tests.stockfishchess.org/api/nn/:nnue1
+DISTFILES=	nn-${_NNUE1_VER}.nnue:nnue1
 EXTRACT_ONLY=	${DISTNAME}${EXTRACT_SUFX}
 
 MAINTAINER=	execve@gmail.com
@@ -27,8 +25,7 @@ MAKE_ARGS=	ARCH=${MYARCH} \
 		PREFIX=${PREFIX}
 ALL_TARGET=	build
 
-_NNUE1_VER=	c288c895ea92
-_NNUE2_VER=	37f18f62d772
+_NNUE1_VER=	1a298aa575a0
 
 BINARY_ALIAS=	wget=true
 
@@ -77,7 +74,6 @@ MYARCH=		general-32
 
 post-patch:
 	@${CP} ${_DISTDIR}/nn-${_NNUE1_VER}.nnue ${WRKSRC}/.
-	@${CP} ${_DISTDIR}/nn-${_NNUE2_VER}.nnue ${WRKSRC}/.
 
 do-install:
 	${INSTALL_PROGRAM} ${WRKSRC}/stockfish ${STAGEDIR}${PREFIX}/bin
diff --git a/games/stockfish/distinfo b/games/stockfish/distinfo
index 9cc8bc1..61ea882 100644
--- a/games/stockfish/distinfo
+++ b/games/stockfish/distinfo
@@ -1,7 +1,5 @@
-TIMESTAMP = 1770188374
-SHA256 (nn-c288c895ea92.nnue) = c288c895ea924429ea9092e3f36b2b3c1f00f2a3a4c759ff7e57e79e3b43e4a7
-SIZE (nn-c288c895ea92.nnue) = 108919594
-SHA256 (nn-37f18f62d772.nnue) = 37f18f62d772f3107e1d6aaca3898c130c3c86f2ab63e6555fbbca20635a899d
-SIZE (nn-37f18f62d772.nnue) = 3519630
-SHA256 (official-stockfish-Stockfish-sf_18_GH0.tar.gz) = 22a195567e3493e7c9ca8bf8fa2339f4ffc876384849ac8a417ff4b919607e7b
-SIZE (official-stockfish-Stockfish-sf_18_GH0.tar.gz) = 251738
+TIMESTAMP = 1788793711
+SHA256 (nn-1a298aa575a0.nnue) = 1a298aa575a085434d29027978dc36867fe9c5bcea9376654b7a8eba1e52dfc2
+SIZE (nn-1a298aa575a0.nnue) = 98511183
+SHA256 (official-stockfish-Stockfish-sf_19_GH0.tar.gz) = 519b653d0d1ffb96531d982ccbe5c6a19425e8388e0e3c2f70f34b424ab32d76
+SIZE (official-stockfish-Stockfish-sf_19_GH0.tar.gz) = 291206
